evalml/pipelines/components/transformers/preprocessing/featuretools.py

```diff
@@ -30,7 +30,6 @@ def __init__(self, index='index', random_state=None, random_seed=0, **kwargs):
         self.features = None
         parameters.update(kwargs)
         super().__init__(parameters=parameters,
-                         random_state=random_state,
```

Comment:
Can you remove the `random_state` documentation here

Reply:
Done!